tests/field_deconstruction/tests.py

```diff
@@ -28,6 +28,27 @@ def test_name(self):
         name, path, args, kwargs = field.deconstruct()
         self.assertEqual(name, "author")
 
+    def test_db_tablespace(self):
+        field = models.Field()
+        _, _, args, kwargs = field.deconstruct()
+        self.assertEqual(args, [])
+        self.assertEqual(kwargs, {})
+        # With a defined DEFAULT_DB_TABLESPACE.
+        with self.settings(DEFAULT_DB_TABLESPACE='foo'):
+            _, _, args, kwargs = field.deconstruct()
+            self.assertEqual(args, [])
+            self.assertEqual(kwargs, {})
+        # With a db_tablespace.
+        field = models.Field(db_tablespace='foo')
+        _, _, args, kwargs = field.deconstruct()
+        self.assertEqual(args, [])
+        self.assertEqual(kwargs, {'db_tablespace': 'foo'})
+        # With a db_tablespace equals to DEFAULT_DB_TABLESPACE.
+        with self.settings(DEFAULT_DB_TABLESPACE='foo'):
+            _, _, args, kwargs = field.deconstruct()
+            self.assertEqual(args, [])
+            self.assertEqual(kwargs, {'db_tablespace': 'foo'})
+
```

Comment:
`db_tablespace` deconstruction ended up being untested so I added a test to make sure I wasn't breaking anything. Note that this also slightly changes the behavior of explicitly passing a `db_tablespace == DEFAULT_DB_TABLESPACE`.